opennmt/experiments/runs/multi_train_a100/config.yaml

```diff
@@ -0,0 +1,221 @@
+seed: 1234
+
+test_src: /dir/test.hu
+test_tgt: /dir/test.en
+
+#------------------------------------------------------------------------------
+# Metadata
+
+_augmentation_description: "Simple training (with augmentation) example"
+
+#------------------------------------------------------------------------------
+# General config
+
+general:
+    # The post fix of the source files (eg. en -> file_name.en)
+    src_postfix: hu
+
+    # The post fix of the target files (eg. en -> file_name.en)
+    tgt_postfix: en
+
+
+#------------------------------------------------------------------------------
+# Multi train config
+
+# train multiple models at the same time with different data
+multi_train:
+    active: true
+
+    generate_configs: true
+
+    # directory where the preprocessed data wil be created
+    data_directory: data/iwslt/iwslt-en-de/
+
+    backwards: true
```

Comment:
I don't really understand what this does.

Reply:
Swapping the source-target languages. Let me know if you have a better name for it

Reply:
Should be smth like train_both_way